libr/debug/debug.c

```diff
@@ -1535,12 +1523,14 @@ R_API ut64 r_debug_get_baddr(RDebug *dbg, const char *file) {
 		return 0LL;
 	}
 #if __WINDOWS__
-#pragma message ("KILL ME PLS")
-	void *foo = dbg->iob.io->cb_printf;
-	dbg->iob.io->cb_printf = __winbase_cb_printf;
-	dbg->iob.system (dbg->iob.io, "winbase");
-	dbg->iob.io->cb_printf = foo;
-	return r_num_get (NULL, winbase_str);
+	typedef struct {
```

Comment:
this is as ugly as the previous way to solve it

Reply:
im thinking about add a get base callback at io but i dont want break nothing at moment this solve the debugger issue, and i agree is ugly but is more fast and dont need a io system callback.